pkg/lib/elasticsearch/service.go

```diff
@@ -165,6 +150,24 @@ func (s *Service) getSource(userID string) (*model.ElasticsearchUserSource, acti
 		if preferredUsername, ok := claims[model.ClaimPreferredUsername]; ok {
 			raw.PreferredUsername = append(raw.PreferredUsername, preferredUsername)
 		}
+		switch identityInfo.Type {
+		case model.IdentityTypeOAuth:
+			raw.OAuthSubjectID = append(raw.OAuthSubjectID, identityInfo.OAuth.ProviderSubjectID)
+		case model.IdentityTypeLoginID:
+			// No additional fields
+		case model.IdentityTypeAnonymous:
+			// No additional fields
+		case model.IdentityTypeBiometric:
+			// No additional fields
+		case model.IdentityTypePasskey:
+			// No additional fields
+		case model.IdentityTypeSIWE:
+			// No additional fields
+		case model.IdentityTypeLDAP:
+			// No additional fields
+		default:
+			panic("identity: unknown identity type: " + identityInfo.Type)
```

Comment:
I think the convention is prefix the panic message with the package name. Here is not `identity: `